ml-agents-envs/mlagents/envs/sampler_class.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+from functools import *
+from abc import ABC, abstractmethod
+
+from .exception import SamplerException
+
+class SamplerException(Exception):
```

Comment:
BTW this should inherit from `UnityException` as with all of the other exception classes

Reply:
Instance of dead code I was using for testing; removed in the next update